### DISCUSSION TITLE
TF-IDF matrix

### DISCUSSION THREAD

**Discussion by fl-yu** [GreenleafLab/ArchR#961] [Questions / Documentation]
Dear team,

How to get the TF-IDF matrix (for tiles or peaks) that is used for iterative LSI? I go through the tutorial and did not find such information. Thanks!

**Comment by rcorces** [comment]
Please do not post usage questions to the Issues section. This is clearly stated in the Issue template instructions. I am transferring this to Discussions.
![image](https://user-images.githubusercontent.com/10343046/128410772-dac36c29-f527-4302-ba27-23221baf9341.png)

**Comment by rcorces** [comment]
The I dont believe that the TF-IDF matrix is stored anywhere, just the SVD results in reducedDims$IterativeLSI
